test/grpcbox_SUITE.erl

```diff
@@ -310,7 +320,7 @@ end_per_testcase(unary_no_auth, _Config) ->
     ok;
 end_per_testcase(multiple_servers, _Config) ->
     ok;
-end_per_testcase(unary_garbage_collect_streams, _Config) ->
+end_per_testcase(fd_socket_option, _Config) ->
```

Comment:
Was about to merge this when I saw the end for testcase `unary_garbage_collect_streams` was removed, I take it that was on accident? :) Can you add it back and I'll merge.